.changeset/eleven-deers-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@codecov/standalone-analyzer": patch
+---
+
+Add support for no-bundler through new Standalone Analyzer library and CLI
```

Comment:
changeset will be committed once the `prerelease` branch is ready to merge to `main`